src/packages/application/initialize.js

```diff
@@ -20,7 +20,8 @@ export default async function initialize<T: Application>(app: T, {
   path,
   port,
   logging,
-  database
+  database,
+  server: serverOpts
```

Comment:
I don't love how here and below I have to rename the key to serverOpts, but we define server elsewhere. I'm open to suggestions!